### ISSUE TITLE
Querybuilder For with selectInserted set to true throws InvalidOperationException

### ISSUE THREAD

**Issue by Syzuna** [edgedb/edgedb-net#6]
**Summary**
When trying to do a bulk insert with a for and asking to select the inserted objects the query builder throws an InvalidOperationException because it cant wrap a global query.

**Code**

```csharp
        return await QueryBuilder
            .For(factions, faction => QueryBuilder.Insert(faction, true))
            .ExecuteAsync(_edgeDb, token: cancellationToken);
```

**Stacktrace**
```
System.InvalidOperationException: Cannot wrap a global-defined query    at EdgeDB.QueryNodes.SelectNode.FinalizeQuery()    at EdgeDB.QueryNodes.ForNode.<FinalizeQuery>b__3_0(QueryNode x)    at System.Linq.Enumerable.SelectListIterator`2.MoveNext()    at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()    at System.Linq.Enumerable.Aggregate[TSource](IEnumerable`1 source, Func`3 func)    at EdgeDB.QueryNodes.ForNode.FinalizeQuery()    at EdgeDB.QueryBuilder`2.InternalBuild(Boolean includeGlobalsInQuery, Action`1 preFinalizerModifier)    at EdgeDB.QueryBuilder`2.IntrospectAndBuildAsync(IEdgeDBQueryable edgedb, CancellationToken token)    at EdgeDB.QueryBuilder`2.EdgeDB.Interfaces.IMultiCardinalityExecutable<TType>.ExecuteAsync(IEdgeDBQueryable edgedb, Nullable`1 capabilities, CancellationToken token)    at SyzunaPrograms.ED.EDCIA.DataImport.Data.Repositories.FactionRepository.InsertFactionsAsync(Faction[] factions, CancellationToken cancellationToken)
```

Original issue report in old archived repo: https://github.com/quinchs/EdgeDB.Net/issues/33

**Comment by quinchs** [comment]
Seems to be fixed in most recent version of the query builder, can you confirm @Syzuna?

**Comment by Syzuna** [comment]
Cannot retest bcs I ran into another issue now:

```
EdgeDB[10] Error occured while reading binary stream System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.    at EdgeDB.Binary.ClientPacketDuplexer.ReadExactAsync(Memory`1 buffer, CancellationToken token)    at EdgeDB.Binary.ClientPacketDuplexer.ReadAsync()
```

**Comment by quinchs** [comment]
That should be fixed in 1.0.1 of the driver and [>=build 63](https://www.myget.org/feed/edgedb-net/package/nuget/EdgeDB.Net.QueryBuilder/1.0.1-build-63) of the query builder

**Comment by Syzuna** [comment]
This seems fixed yeah
